server/server.go

```diff
@@ -31,6 +31,7 @@ type Server struct {
 	initializers      []InitializerFunc
 	initializeTimeout time.Duration
 	registrars        []func(mux *http.ServeMux) error
+	middleware        Middleware
```

Comment:
From my point of view field "middleware" should be repeated to allow use to define chain of middleware, solution with one middleware from my point of view isn't enough flexible